eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -215,7 +215,7 @@ stages:
                     steps:
                       - checkout: self
                       - pwsh: |
-                          eng/scripts/Update-PkgVersion.ps1 -ServiceDirectory '${{parameters.ServiceDirectory}}' -PackageName '${{artifact.name}}' -PackageDirName '${{artifact.directoryName}}'
```

Comment:
Can we remove artifact.directoyrName from the repo now as well or is it used for other things?

Reply:
I think we still use it for some other things. e.g. in the github.io doc generation process.

Reply:
OK we can keep it for now but eventually I'd like to get the other process using the same properties.